test/resources/accounts-acl/localhost/.acl

```diff
@@ -0,0 +1,10 @@
+# Root ACL resource for the root
```

Comment:
I suppose these were generated by the test, so I would think that we shouldn't commit them, but rather have the test clean up after itself?

Reply:
@kjetilk I actually left this in on purpose, since there is also an index.html and index.html.acl file in this folder.
Should I remove those as well? Or just leave it as-is?

Reply:
Hmmm, good point. I guess it depends on how they are updated, if they are updated by the test script and that will overwrite any changes to it, then it shouldn't be committed.

Reply:
Here's a list of the new .acl files + reason:
* `test/resources/accounts-acl/config/templates/server/.acl`: generated, like all other files in this config folder. (if we delete this, we should probably delete this whole config folder)
* `test/resources/accounts-acl/localhost/.acl`: needed for the unit tests, like the other files in this dir
* `test/resources/accounts/localhost/api/.acl`: needed for the unit tests
* `test/resources/headers/.acl`: needed for the unit tests, like the other files in this dir

Reply:
OK, sounds good. And besides, not much can go wrong.